docker/prognostic_run/Dockerfile

```diff
@@ -73,23 +73,30 @@ RUN cd /tmp/fortran-build/FV3 && \
     make -j 8 && \
     PREFIX=/usr/local make install
 
+# gcloud
+
+#Install gcloud
+RUN apt-get update && apt-get install -y  apt-transport-https ca-certificates gnupg curl gettext && \
+    echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list &&\
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - && \
+    apt-get update && apt-get install -y google-cloud-sdk jq python3-dev python3-pip kubectl gfortran graphviz
+
+
 #
 # Python Stuff Here
 #
-
-# Python components here
-COPY docker/prognostic_run/requirements/base.txt /tmp/requirements.txt
-COPY constraints.txt /tmp/constraints.txt
```

Comment:
this also speeds up the build...only the relevant changes to the constraints will cause a problem.